app/models/internal/impl/DefaultExpressionQuery.java

```diff
@@ -152,6 +170,7 @@ public String toString() {
     }
 
     private final ExpressionRepository _repository;
+    private Organization _organization;
```

Comment:
So we can make this final if it's accepted via the constructor and via the repository createQuery API. That's probably the simplest way forward. Refactoring to a builder model would work too. If there will be other required fields sticking with this pattern will eventually mean undoing it. However, I'm not sure I can see a lot (any?) other required fields on the query.